.rubocop.yml

```diff
@@ -12,7 +10,11 @@ AllCops:
   Exclude:
     - 'db/**/*'
     - 'bin/**/*'
-    - 'config/**/*'
+    - 'config/clamd/*'
```

Comment:
Probably simpler to just do

```yaml
Include:
  - 'config/**/*.rb'
```

Reply:
That would still include some non .rb files. Decided to use a more catch all approach instead as I've seen you can do regex in this, so excluding anything that doesn't end in .rb

Reply:
Nope it'll just glob all the ruby files

![image](https://github.com/user-attachments/assets/9f69e85d-b742-4d8f-aadf-1910e0fade2c)

Reply:
there's non .rb files at config/* also wouldn't those get included?

Reply:
No, the current glob does but the above glob only finds `.rb` files. See above screenshot 😄

Reply:
ah fair enough - I must have been reading the syntax wrong. Still thinking the regex might be better though since this means whatever we put in, wherever it is, if it's a ruby file - we rubocop it